.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -55,8 +63,9 @@ New features include {{key_features}}. We've also implemented loads of other imp
 
 Read the [release notes]({{url}}) for full details.
 
-Following our support policy, versions {{versions}} are no longer supported. Projects running these versions should start planning to upgrade in the near future. For more details read our [software support documentation](https://docs.communityhealthtoolkit.org/core/overview/supported-software/).
+Following our support policy, versions {{versions}} are no longer supported. Projects running these versions should start planning to upgrade in the near future. For more details read our [software support documentation](https://docs.communityhealthtoolkit.org/core/releases/#supported-versions).
 
 Check out our [roadmap](https://github.com/orgs/medic/projects/112) to see what we're working on next.
 ```
+- [ ] Add one last update to the [Medic Product Team call agenda](https://docs.google.com/document/d/14AuJ7SerLuOPESBjQlJqpBtzwSAoVf5ykTT7fjyJBT0/edit) and lead a release retrospective covering what went well and areas to improve for next time.
```

Comment:
I definitely think this should be an internal retro - it's mostly about improving internal process.

Personally I think it fits well within the product team call because it's basically the same people, fits well with a sort of celebration (what went well), and means you don't have to schedule a new meeting that won't suit all timezones.

The main question is, do you think we have ~30 minutes to spare in the team call for this?

Reply:
I agree that an internal retro is useful, and we can certainly make time for this on our call. 

We should _also_ do a more community focused retro as well, which would likely come at a later time and coordinated by @smbuthia. We can add details for that later.